tests/cross/src/test/scala/tests/highlight/DocumentHighlightSuite.scala

```diff
@@ -699,4 +699,11 @@ class DocumentHighlightSuite extends BaseDocumentHighlightSuite {
        |}""".stripMargin,
   )
 
+  check(
+    "select-parentheses",
+    """|object Main {
+       |  val a = (1 + 2 + 3).<<toStr@@ing>>
+       |}""".stripMargin,
+  )
+
```

Comment:
Let's also add:
```scala
  check(
    "select-parentheses2",
    """|object Main {
       |  val a = (1 + 2 + 3) <<:@@:>> Nil
       |}""".stripMargin,
  )
```

Reply:
It was worth adding this test, now I know it doesn't work for scala 2 😅 . I will take a look on that in a follow up, the issue might be a little bit more complicated, because right associative operator is translated to block with synthetic symbol `$rightassoc1`

Reply:
Uch, yeah let's do a follow up and ignore for now